lib/apiv2/workloadendpoint.go

```diff
@@ -42,7 +42,9 @@ type WorkloadEndpointSpec struct {
 	// The container ID.
 	ContainerID string `json:"containerID,omitempty" validate:"omitempty,name"`
 	// The Pod name.
-	Pod string `json:"podID,omitempty" validate:"omitempty,name"`
+	Pod string `json:"pod,omitempty" validate:"omitempty,name"`
+	// The Endpoint name.
```

Comment:
can you add one more sentence in the comment explaining this is the container side of the veth pair interface?

Reply:
it doesn't have to be.

Reply:
in libnetwork it's an arbitrary ID.

Reply:
K